### PR TITLE
Summary UI fix

### LINKED ISSUE #74: Fix Summary UI
Make the look of the Summary page look better in the create experiment sequence.

### PR DESCRIPTION
**Pull request by coridiaz**
**What Was Changed?**
Summary UI page formatting was updated 

**Why Was This Changed?**
The summary page for new experiment setup did not always format properly before.
It now clearly labels the data displayed and is spaced properly for better user experience.

**Screenshots:**
Screenshot 2023-03-07 164534

Closes #74

### CODE REVIEW

File: experiment_pages/summary_ui.py [hunk 1]
```diff
@@ -26,56 +26,85 @@ def __init__(self, input: Experiment, parent:Tk, prev_page: Frame, menu_page: Fr
...
+        labels.append(name_label)
+        inputs.append(name_input)
 
         names = ''
         for name in self.input.get_investigators():
-            names += name + ', '
-        Label(self.main_frame, text=names).grid(row=1, column=0, padx=10, pady=10)
+            names += name + ',\n'
+        invest_label = Label(self.main_frame, text='Investigators:', style='Summary.TLabel')
+        invest_input = Label(self.main_frame, text=names[:-2])
```
Comment (aithaprasad) [inline experiment_pages/summary_ui.py:57]: Add a if condition to check whether the names list is > 2.

File: experiment_pages/summary_ui.py [hunk 2]
```diff
@@ -26,56 +26,85 @@ def __init__(self, input: Experiment, parent:Tk, prev_page: Frame, menu_page: Fr
...
+        labels.append(species_label)
+        inputs.append(species_input)
 
         items = ''
         for item in self.input.get_measurement_items():
-            items += item + ', '
-        Label(self.main_frame, text=items).grid(row=2, column=0, padx=10, pady=10)
+            items += item + ',\n'
+        items_label = Label(self.main_frame, text='Measurement Items:', style='Summary.TLabel')
+        items_input = Label(self.main_frame, text=items[:-2])
```
Comment (aithaprasad) [inline experiment_pages/summary_ui.py:70]: sam if condition here too.

File: experiment_pages/summary_ui.py [hunk 3]
```diff
@@ -26,56 +26,85 @@ def __init__(self, input: Experiment, parent:Tk, prev_page: Frame, menu_page: Fr
...
         groups = self.input.get_group_names()
-        index = 0
+        group_names = ''
         for group in groups:
-            Label(self.group_frame, text=group).grid(row=index, column=0, padx=10, pady=10)
-            index += 1
+            group_names += group + ',\n'
+            
+        group_label = Label(self.main_frame, text='Group Names:', style='Summary.TLabel')
+        group_input = Label(self.main_frame, text=group_names[:-2])
```
Comment (aithaprasad) [inline experiment_pages/summary_ui.py:91]: same if condition here too.

File: experiment_pages/summary_ui.py (hunk 1 above)
Comment (aithaprasad) [reply experiment_pages/summary_ui.py:57]: Never expect the input is always as we need.

Comment (coridiaz) [reply experiment_pages/summary_ui.py:57]: the -2 is the second to last char in the investigator name string I am adding as the "," and new line. This line was put in there to remove the comma from the last item in a list. 
Would you still like a conditional?

Comment (aithaprasad) [reply experiment_pages/summary_ui.py:57]: Yes,  by chance if get_investigators is empty then your names would be empty.

Comment (coridiaz) [reply experiment_pages/summary_ui.py]: ok done!

File: experiment_pages/summary_ui.py (hunk 2 above)
Comment (coridiaz) [reply experiment_pages/summary_ui.py]: done!

File: experiment_pages/summary_ui.py (hunk 3 above)
Comment (coridiaz) [reply experiment_pages/summary_ui.py]: done!

**Review by aithaprasad — APPROVED** [review]

**Pull request merged**